### PR TITLE
Found and fixed some typo 

### PR DESCRIPTION
**Pull request by prajesh007**
As per my knowledge , I tried to fixed some typing errors

### CODE REVIEW

File: CHAOSScon/2019NA/desc.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## About CHAOSScon
 
-Learn about open source project health metrics and tools used by open source projects, communities, and engineering teams to track and analyze their community work. This conference will provide a venue for discussing open source project health, CHAOSS updates, use cases, and hands-on workshops for developers, community managers, project managers, and anyone interested in measuring open source project health. We will also share insights from the CHAOSS working groups on *Diversity and Inclusion*, *Evolution*, *Risk*, *Value*, and *Common Metrics*.
+Learn about open source project health metrics, and tools used by open source projects, communities, and engineering teams to track and analyze their community work. This conference will provide a venue for discussing open source project health, CHAOSS updates, use cases, and hands-on workshops for developers, community managers, project managers, and anyone interested in measuring open source project health. We will also share insights from the CHAOSS working groups on *Diversity and Inclusion*, *Evolution*, *Risk*, *Value*, and *Common Metrics*.
```
Comment (germonprez) [inline CHAOSScon/2019NA/desc.md:3]: The original is fine.

File: CHAOSScon/2019NA/speakers.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ As a software engineer for Indeed's open source team, Dani builds tools to help
 
 Before learning to code, Dani worked at non-profits and was a professional backpacking guide. She became passionate about social justice, interconnectedness, and sharing good ideas, and now brings those passions to her work as a technologist. 
 
-Dani is also an educator and experienced speaker who has given talks at Waffle.js, SCaLE, and more. She is always interested in talking about open source sustainability, learning to code, or your favorite place to go hiking.
+Dani is also an educator and experienced speaker who has given talks at Waffle.js, SCALE and more. She is always interested in talking about open source sustainability, learning to code, or your favorite place to go hiking.
```
Comment (germonprez) [inline CHAOSScon/2019NA/speakers.md:66]: Original is correct

**Review by germonprez — COMMENTED** [review]
Please update the PR with the comments.

**Comment by klumb** [comment]
No response so I am closing

**Pull request closed without merging**